Makefile

```diff
@@ -156,14 +156,18 @@ delete-test-cluster:
 mlflow-e2e:
 	@./scripts/ci/mlflow-e2e.sh
 
+mlflow-seldon-e2e:
+	@./scripts/ci/mlflow-seldon-e2e.sh
+
 fuseml-install:
 	@./dist/fuseml-installer install
 
 fuseml-install-with-extensions:
-	@./dist/fuseml-installer install --extensions mlflow,kfserving
+	@./dist/fuseml-installer install --extensions mlflow,kfserving,seldon-core
```

Comment:
I also recommend creating a new make target where it installs only mlflow and seldon-core extensions and use that on the test. So, in case of kfserving failure it won't affect seldon-core tests.